models/palm_detection_mediapipe/demo.py

```diff
@@ -69,19 +69,24 @@ def visualize(image, score, palm_box, palm_landmarks, fps=None):
         image = cv.imread(args.input)
 
         # Inference
-        score, palm_box, palm_landmarks = model.infer(image)
-        if score is None or palm_box is None or palm_landmarks is None:
+        results = model.infer(image)
+        if results is None:
```

Comment:
if `results` is an empty array, we do not need this if condition and the for loop stops.